src/Pact/Types/ECDSA.hs

```diff
@@ -1,150 +0,0 @@
-{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
If we're keeping ETH we presumably want to keep this module, although the pattern for WebAuthn seems to keep formats private to `Pact.Types.Crypto` so maybe it's just moving relevant stuff there. If we do keep this module we should consider moving `WebAuthnSignature` to its own module for consistency.